pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -7,25 +7,16 @@ SPDX-License-Identifier: Apache-2.0
 package didexchange
 
 import (
+	"time"
+
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/decorator"
 	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 )
 
-// CreateInvitationResponse model
-//
-// This is used for returning a create invitation response with a single connection invitation as body
-//
-// swagger:response createInvitationResponse
-type CreateInvitationResponse struct {
-	// in: body
-	Invitation *InvitationRequest `json:""`
-}
-
 // InvitationRequest model
 //
 // This is used for returning a create invitation response with a single connection invitation as body
 //
-// swagger:model InvitationRequest
 type InvitationRequest struct {
```

Comment:
Please also remove `InvitationRequest` from this package

Reply:
Removed, it came while resolving conflict